confgenerator/config.go

```diff
@@ -99,6 +99,8 @@ func (ve validationError) Error() string {
 		return fmt.Sprintf("%q must be a duration of at least %s", ve.Field(), ve.Param())
 	case "endswith":
 		return fmt.Sprintf("%q must end with %q", ve.Field(), ve.Param())
+	case "experimental":
+		return experimentalValidationErrorString(ve)
```

Comment:
`"experimental"` comes from the `tag` argument of [`sl.ReportError`](https://github.com/GoogleCloudPlatform/ops-agent/blob/jefferbrecht-otlp/confgenerator/experimental.go#L39) in this case.

I chose to encode the whole error message inside of `ve.Param` because it felt simpler, but I agree maybe it's not great usage of the API. I'll refactor this.